### PR TITLE
Abre ae, eaester egg

### PR DESCRIPTION
**Pull request by JoaoVictorRDSs**
iab9gdwqgdoqudqw
igão guei

### CODE REVIEW

**Comment by JoaoVictorRDSs** [comment]
oi
thcau

File: a1.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 linha 1 arquivo 1
 linha 2 arquivo 1
-linha 3 arquivo 1
\ No newline at end of file
+linha 3 arquivo 1
+ajdowqhd9qdq
+joãozin é muito lindo
```
Comment (igorhgt) [inline a1.txt]: mentira

Comment (JoaoVictorRDSs) [reply a1.txt]: mentira nada

**Comment by igorhgt** [comment]
ta bao jao

File: a1.txt (hunk 1 above)
Comment (JoaoVictorRDSs) [reply a1.txt]: > ta bao jao

eu to bao e vc ta bao

**Pull request merged**